File: LaboratorioDeAlgoritmos/Aula03B/exemplo_es_02.c
/*Escreva um programa em C que calcule sua média semestral. */

#include <stdio.h>

int main()
{
	float np1, np2;
	float trab;
	float ms;

	printf("Calculo da media semestral. ");
	printf("\n\n");

	printf("Informe a primeira nota do professor: ");
	scanf("%f", &np1);

	printf("Informe a segunda nota do professor: ");
	scanf("%f", &np2);

	printf("Informe a nota do trabalho: ");
	scanf("%f", &trab);

	ms = (np1 * 4 + trab * 2 + np2 * 4) / 10;

	printf("Media semestral: %2.2f", ms);
	printf("\n\n");

	system("PAUSE");
	return 0;
}

File: LaboratorioDeAlgoritmos/Aula03B/exemplo_es_02.py
def main():
    print("Calculo da média semestral\n\n")

    np1 = int(input("Informe a primeira nota do professor: "))

    np2 = int(input("Informe a primeira nota do professor: "))

    trab = int(input("Informe a nota do trabalho: "))

    ms = (np1 * 4 + trab * 2 + np2 * 4) / 10

    print("Média semestral: %2.2f." % (ms))


if __name__ == '__main__':
    main()
